tests/docker/Dockerfile

```diff
@@ -62,8 +62,12 @@ LABEL ducker.creator=$ducker_creator
 # Update Linux and install necessary utilities.
 # we have to install git since it is included in openjdk:8 but not openjdk:11
 RUN apt update && apt install -y sudo git netcat iptables rsync unzip wget curl jq coreutils openssh-server net-tools vim python3-pip python3-dev libffi-dev libssl-dev cmake pkg-config libfuse-dev iperf traceroute iproute2 iputils-ping && apt-get -y clean
-RUN python3 -m pip install -U pip==21.1.1;
-RUN pip3 install --upgrade cffi virtualenv pyasn1 boto3 pycrypto pywinrm ipaddress enum34 debugpy && pip3 install --upgrade "ducktape>0.8"
+RUN apt install software-properties-common -y
```

Comment:
the installed version of python in this image is `3.9.2`, so why we need those changes?
```
ducker@6bbe07c680d4:/$ python3 --version
Python 3.9.2
```

Reply:
Sorry, I got this issue mixed up with other issues.
Reverted.